src/NServiceBus.ServicePlatform.Connector.UnitTests/NServiceBus.ServicePlatform.Connector.UnitTests.csproj

```diff
@@ -17,7 +17,7 @@
     <PackageReference Include="NUnit3TestAdapter" Version="4.1.0" />
     <PackageReference Include="Particular.Approvals" Version="0.3.0" />
     <PackageReference Include="PublicApiGenerator" Version="10.2.0" />
-    <PackageReference Include="NServiceBus.Extensions.Hosting" Version="1.1.0" />
+    <PackageReference Include="NServiceBus.Extensions.Hosting" Version="[2.0.0-alpha.153, 3.0.0)" />
```

Comment:
I think the one in the test package is supposed to be a specific version and not a range. This is what will allow Dependabot to update the dependency in the repo when a new version of hosting is released.

Reply:
Set to a specific version.